modules/ocf_ceph/manifests/mon.pp

```diff
@@ -0,0 +1,62 @@
+# Ceph monitor + manager + metadata server configuration
+
+class ocf_ceph::mon {
+  require ocf_ceph
+  require ocf_ceph::admin
+
+  $mon_key = lookup('ceph::mon_key')
+
+  $mgr_perms = "mon 'allow profile mgr' osd 'allow *' mds 'allow *'"
+  $mgr_keyring_file = "/var/lib/ceph/mgr/ceph-${::hostname}/keyring"
+
+  $mds_perms = "mon 'allow profile mds' osd 'allow *' mds 'allow *'"
```

Comment:
why is this a variable if it's being used once in the exec. this is really odd.

Reply:
Otherwise the exec gets too long